src/null-safety/faq.md

```diff
@@ -301,6 +301,77 @@ and situations where a null value is really expected. So the tool tells you what
 it knows ("it looks like this condition will always be false!") and lets you
 decide what to do.
 
+## What should I know about dart2js and null safety?
+
+For a long time, the dart2js compiler has had optimizations specifically
```

Comment:
Maybe having a link to [dart2js](https://dart.dev/tools/dart2js) somewhere would be worthwhile?